pages/index.tsx

```diff
@@ -1,9 +1,34 @@
 import type { NextPage } from "next";
 import { ThemeProvider } from "@mui/material";
 import { theme } from "../styles/theme";
+import { Divider } from "@mui/material";
+import { Typography } from "@mui/material";
+import Link from "@mui/material/Link";
 
 const Home: NextPage = () => {
-  return <ThemeProvider theme={theme}></ThemeProvider>;
+  return (
+    <ThemeProvider theme={theme}>
+      <div className="tool-box">
+        <h3>Tools</h3>
```

Comment:
Since we're using Typography everywhere else let's use it here too for consistency <Typography variant="h3">